src/System.Xml.XmlSerializer/pkg/System.Xml.XmlSerializer.pkgproj

```diff
@@ -1,13 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Version>4.0.12.0</Version>
+    <Version>4.1.0.0</Version>
```

Comment:
@weshaggard Jason pointed out that the XmlSerializer package's version had been changed to [4.3.0.XXXX](https://dotnet.myget.org/feed/dotnet-core/package/nuget/System.Xml.XmlSerializer). Do we still need to bump the package's version in dev/api branch?